stubstub/activity.ts

```diff
@@ -29,16 +28,26 @@ export const expectCreatedActivityItem = <Result extends Record<string, any>>(
 ): ExpectCreatedActivityItemResult<Result> => {
 	const cutoffTime = new Date();
 	return {
-		toMatchObject: async (values) => {
+		toMatchResultingObject: async (responseFn) => {
 			const result = await promise;
 			await finishDeferredTasks();
-			const resolvedValues = typeof values === 'function' ? values(result) : values;
+			const resolvedValues = responseFn(result);
 			const { query, assertion } = splitQueryFromAssertion(resolvedValues);
 			const item = await ActivityItem.findOne({
 				where: { ...query, createdAt: { [Op.gte]: cutoffTime } },
 			});
 			expect(item).toMatchObject(assertion);
 			return result;
 		},
+		toMatchObject: async (values) => {
+			const result = await promise;
+			await finishDeferredTasks();
+			const { query, assertion } = splitQueryFromAssertion(values);
+			const item = await ActivityItem.findOne({
+				where: { ...query, createdAt: { [Op.gte]: cutoffTime } },
+			});
+			expect(item).toMatchObject(assertion);
+			return result;
+		},
```

Comment:
Was splitting this function in two a matter of preference (e.g. to keep `toMatchObject` closer to Jest's version), or was it necessary for some other reason?